mis/data/graphs.py

```diff
@@ -538,3 +540,39 @@ def ingest(self, graph: NXWithPos, device: pl.devices.Device, id: int) -> BaseGr
         pyg.pos = torch.tensor(positions, dtype=torch.float)
 
         return BaseGraph(id=id, device=device, data=pyg)
+
+
+@dataclass
+class DIMACSDataset:
```

Comment:
Let's take the opportunity to add some documentation here.

Reply:
Ok done